app/src/features/observations/BlockObservationPage.tsx

```diff
@@ -189,7 +197,7 @@ const BlockObservationPage = () => {
     return <CircularProgress className="pageProgress" size={40} />;
   }
 
-  const handleSaveAndExit = async () => {
+  const handleSaveAndExit = async (action: string) => {
```

Comment:
Do we need this `action` param? Or lower down when we make the api call, can't we determine which to call based on the existence of the observationId?

If that isn't possible for some reason, then we should at least make this string an enum so we only get passed known options.

Reply:
no we definitely can :) good one